diff --git a/gnome/Makefile.am b/gnome/Makefile.am
index 662c6c7..ded4c39 100644
--- a/gnome/Makefile.am
+++ b/gnome/Makefile.am
@@ -18,7 +18,7 @@ pkgconfig_DATA = $(PKG_CONFIG_FILES)
 lib_LTLIBRARIES = $(GNOME_TARGETS)
 
 loaderdir = $(GDK_PIXBUF_DIR)
-loader_LTLIBRARIES = $(GDK_PIXBUF_PLUGIN)
+#loader_LTLIBRARIES = $(GDK_PIXBUF_PLUGIN)
 
 libopenrawgnome_la_LDFLAGS = \
 	-version-info @LIBOPENRAW_VERSION_INFO@
